gbpservice/nfp/orchestrator/config_drivers/heat_driver.py

```diff
@@ -1071,6 +1420,37 @@ def is_config_complete(self, stack_id, tenant_id,
             LOG.exception(_LE("Retrieving the stack %(stack)s failed."),
                           {'stack': stack_id})
             return failure_status
+    
+    def check_config_complete(self, nfp_context):
+        provider_tenant_id = nfp_context['tenant_id']
+
+        heatclient = self._get_heat_client_v1(provider_tenant_id)
+        if not heatclient:
+            return failure_status
+        try:
```

Comment:
@ashutosh-mishra  - it is under testing.. there could be merge issues. We decided to review paralelly while testing is happening so the focus is on logic changes.